heron/cli/src/python/cli_helper.py

```diff
@@ -72,7 +72,7 @@ def run(command, parser, cl_args, unknown_args, action):
         "--command", command,
     ]
 
-    if opts.verbose():
+    if Log.getEffectiveLevel() == logging.DEBUG:
```

Comment:
this is basically about propagating logging level from Python executor down to Java scheduler.